http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -649,6 +650,19 @@ public HttpClientConfig setTcpKeepAliveIdleTime(Duration tcpKeepAliveIdleTime)
         return this;
     }
 
+    @NotNull
```

Comment:
This isn't needed, as the code in this class ensures that the field is not null.